gapic/src/main/com/google/gapid/perfetto/models/CounterTrack.java

```diff
@@ -295,7 +313,7 @@ public Values combine(Values other) {
     }
 
     @Override
-    public Selection<Values.Key> build() {
+    public Selection<Long> build() {
```

Comment:
Delete the Key class below this function if its not getting used anywhere.

Reply:
Thanks for reminding, forgot this bit! Done in the new change.